src/components/sbb-link/sbb-link.scss

```diff
@@ -81,15 +80,15 @@
   }
 }
 
-.sbb-link__icon {
+sbb-icon {
   width: var(--sbb-size-icon-ui-small);
   height: var(--sbb-size-icon-ui-small);
```

Comment:
Is setting width and height really needed? Icon should have the right size itself.